docker-compose.yml

```diff
@@ -335,6 +390,22 @@ services:
     volumes:
       - "sentry-symbolicator:/data"
   web:
+    <<: *sentry_defaults-feature-complete
+    ulimits:
+      nofile:
+        soft: 4096
+        hard: 4096
+    healthcheck:
+      <<: *healthcheck_defaults
+      test:
+        - "CMD"
+        - "/bin/bash"
+        - "-c"
+        # Courtesy of https://unix.stackexchange.com/a/234089/108960
+        - 'exec 3<>/dev/tcp/127.0.0.1/9000 && echo -e "GET /_health/ HTTP/1.1\r\nhost: 127.0.0.1\r\n\r\n" >&3 && grep ok -s -m 1 <&3'
+    profiles:
+      - feature-complete
+  web-errors-only:
```

Comment:
Since we don't modify the `nginx.conf` (or maybe we don't... yet), should we specify `container_name` to have the same value as the usual `web` container? https://docs.docker.com/compose/compose-file/05-services/#container_name

Reply:
Tbh, I don't know if it'll work, but should we give it a try?

Reply:
I considered this, but the default naming of containers also uses the `COMPOSE_PROJECT_NAME`, and I didn't want to overwrite that for our users.

Reply:
hm this will be a problem with both the install script and nginx conf, it would be good to use the same service name

Reply:
I just cleaned this up now :)